controllers/api/tweetController.js

```diff
@@ -0,0 +1,211 @@
+const db = require('../../models')
+const helpers = require('../../_helpers')
+
+const { Tweet, Reply, Like } = db
+
+// @todo - add error handling
+
+const tweetController = {
+  getTweets: async (req, res) => {
+    /* #swagger.tags = ['Tweet']
+        #swagger.description = '瀏覽全部tweets'
+        #swagger.responses[200] = {
+          description: '回傳陣列帶有多個tweet物件',
+          schema: [{"$ref": "#/definitions/Tweet"}]
+        }
+    */
+    try {
+      const tweets = await Tweet.findAll({
+        include: [Reply, Like]
+      })
+      return res.status(200).json(tweets)
```

Comment:
Check array:
```
if (!tweets.length) {
        return res.status(400).json({ status: 'error', message: 'cannot find any tweet' })
      }
```
Do we need isLiked? Frontend can tell by checking Likes array

Reply:
same: empty array needs to be sent, since there is no error during fetching.

But front end does not have access to helpers.getUser(req) to compare.
I think it is better that we do the job because sending helpers.getUser(req) to front-end does not sound logic.
adding isMyTweet is also required for distinguish tweets of user logging in from other tweets.
My code:
`likedTweets = likedTweets.map(like => {
        const tweet = like.Tweet.dataValues
        tweet.isLiked = helpers.getUser(req).Likes.map(myLike => myLike.TweetId).includes(tweet.id)
        tweet.isMyTweet = helpers.getUser(req).id === tweet.User.id
        //discard the like data part
        return tweet
      })`

Reply:
A33: 同儕互評程式碼品質 

描述這組程式碼變動的目的:
getTweets 是要抓取使用者的主頁推文資料，應包含自己及追蹤中的推文，推文要能顯示愛心數跟回覆數量，喜歡過的貼文要能有條件讓前端渲染

✅ 這筆程式碼改動是必須的，下列幾項應注意:
1. 每個推文的user 資料也要抓出來
2. reply 及 like 數可用 Sequelize  直接在資料庫中算好再拿回來，減少資料傳輸量
3. 現在的寫法式拿到所有推文，應該用 helpers.getUser 中的追蹤名單當成條件，只抓取追蹤中的跟自己的推文
4. 加入 isLiked 及 isMyTweet，讓前端可以有條件進行渲染，或未來針對自己推文跟別人推文有不同權限
✅ 命名適當
✅ 註解明確且解釋了程式碼的目的
✅ Coding style 一致